package.json

```diff
@@ -23,31 +23,31 @@
     "projen": "npx projen"
   },
   "devDependencies": {
-    "@types/jest": "^27.5.1",
+    "@types/jest": "^27.5.2",
     "@types/node": "^12",
     "@typescript-eslint/eslint-plugin": "^5",
     "@typescript-eslint/parser": "^5",
     "eslint": "^8",
-    "eslint-import-resolver-node": "^0.3.6",
+    "eslint-import-resolver-node": "^0.3.9",
     "eslint-import-resolver-typescript": "^2.7.1",
-    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-import": "^2.28.1",
     "jest": "^27.5.1",
     "jest-junit": "^13",
     "json-schema": "^0.4.0",
     "npm-check-updates": "^12",
     "projen": "^0.52.37",
     "standard-version": "^9",
     "ts-jest": "^27.1.5",
-    "typescript": "^4.7.2"
+    "typescript": "^4.9.5"
   },
   "dependencies": {
-    "@aws-sdk/client-dynamodb": "^3.100.0",
-    "@aws-sdk/lib-dynamodb": "^3.100.0",
-    "@aws-sdk/smithy-client": "^3.99.0",
-    "@aws-sdk/types": "^3.78.0",
-    "@aws-sdk/util-dynamodb": "^3.100.0",
-    "@types/aws-lambda": "^8.10.97",
-    "aws-sdk": "^2.1145.0"
+    "@aws-sdk/client-dynamodb": "^3.429.0",
+    "@aws-sdk/lib-dynamodb": "^3.429.0",
+    "@smithy/smithy-client": "^3.370.0",
```

Comment:
My closed PR diff are followings.

```
    "@aws-sdk/client-dynamodb": "^3.398.0",
    "@aws-sdk/lib-dynamodb": "^3.398.0",
    "@aws-sdk/types": "^3.398.0",
    "@aws-sdk/util-dynamodb": "^3.398.0",
    "@smithy/smithy-client": "^2.0.5",
```

`@smithy/smithy-client` not have major version 3.

Reply:
Now, latest version is `2.1.11`.

Reply:
Did you test compile? Mine blows up

Reply:
Smithy does look wrong - I'll fix that

Reply:
```ts
 FAIL  test/v3-document.test.ts
  ● Test suite failed to run

    test/v3-document.test.ts:53:13 - error TS2339: Property 'key' does not exist on type 'never'.

    53   get.Item?.key;
                   ~~~

 FAIL  test/v3.test.ts
  ● Test suite failed to run

    test/v3.test.ts:43:13 - error TS2339: Property 'key' does not exist on type 'never'.

    43   get.Item?.key;
                   ~~~
    test/v3.test.ts:162:28 - error TS2339: Property 'key' does not exist on type 'never'.

    162   returnAllNew.Attributes?.key?.S;
                                   ~~~
    test/v3.test.ts:179:28 - error TS2339: Property 'key' does not exist on type 'never'.

    179   returnAllOld.Attributes?.key?.S;
                                   ~~~
    test/v3.test.ts:196:32 - error TS2339: Property 'key' does not exist on type 'never'.

    196   returnUpdatedNew.Attributes?.key?.S;
                                       ~~~
    test/v3.test.ts:213:32 - error TS2339: Property 'key' does not exist on type 'never'.

    213   returnUpdatedOld.Attributes?.key?.S;
                                       ~~~
```